docs/microphone.rst

```diff
@@ -0,0 +1,83 @@
+Microphone **V2**
+*****************
+
+.. py::module:: microbit
+
+This object lets you access the on-board microphone available on the **V2**
```

Comment:
Are we using the term "built-in" as well for the microphone?

Reply:
I think we should. Have updated in https://github.com/bbcmicrobit/micropython/pull/692/commits/78bd2418a5e2b22c538b233e3313806f2dc032e5